Makefile

```diff
@@ -328,3 +330,44 @@ install-gateway-api-local:
 .PHONY: uninstall-gateway-api
 uninstall-gateway-api:
 	kubectl delete -f $(GATEWAY_API_CRDS_LOCAL_PATH)
+
+### dev-env:			  Launch development environment
+.PHONY: dev-env
+dev-env: dev-kind-up dev-pack-image
+	helm repo add apisix https://charts.apiseven.com
+	helm repo update
+	helm install apisix apisix/apisix \
+		--set gateway.type=NodePort \
+		--set ingress-controller.enabled=true \
+		--namespace ingress-apisix \
+		--create-namespace
+	kubectl set image deployment/apisix-ingress-controller ingress-controller=$(DEV_REGISTRY)/apache/apisix-ingress-controller:dev --namespace ingress-apisix
+
+### dev-env-reset:        Reset development environment
+.PHONY: dev-env-reset
+dev-env-reset: kind-reset dev-clean-image
+
+### dev-kind-up:              Create a kind cluster with registry for dev
+.PHONY: dev-kind-up
+dev-kind-up:
+	./utils/kind-with-registry.sh "8000"
```

Comment:
The port shouldn't be hard-coded. But as I said, we don't need two registries, so I guess this line will be gone and doesn't affect anything.

Reply:
![img_v2_834dab00-ca6d-40dd-8f39-a122b7172e1g](https://user-images.githubusercontent.com/63330165/231724894-3e5b5a77-62a6-4f09-a5c3-942c2f9d69a2.jpg)